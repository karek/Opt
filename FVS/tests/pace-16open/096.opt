6
39 10 13 17 5 24 
