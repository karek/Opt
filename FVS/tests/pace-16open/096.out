6
12 41 24 13 17 5 
